libs/components/src/lib/side-drawer/side-drawer.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { elementUpdated, fixture, getControlElement } from '@vivid-nx/shared';
+import { POSITION, SideDrawer } from './side-drawer';
+import '.';
+
+const COMPONENT_TAG = 'vwc-side-drawer';
+
+describe('vwc-side-drawer', () => {
+	let element: SideDrawer;
+
+	beforeEach(async () => {
+		element = await fixture(`<${COMPONENT_TAG}>
+								</${COMPONENT_TAG}>`) as SideDrawer;
+	});
+
+	describe('basic', () => {
+		it('initializes as a vwc-side-drawer', async () => {
+			expect(element).toBeInstanceOf(SideDrawer);
+			expect(element.open).toBeFalsy();
+			expect(element.alternate).toBeFalsy();
+			expect(element.hasTopBar).toBeFalsy();
+			expect(element.position).toBeUndefined();
+			expect(element.modal).toBeFalsy();
+		});
+	});
+
+	describe('show', () => {
+		it('should set "open" to true and add "open" class', async () => {
+			const control = getControlElement(element);
+			const doesNotHaveClassOpenBeforeShow = control.classList.contains('control');
+			
+			element.show();
+			await elementUpdated(element);
+			const hasClassOpenAfterShow = control.classList.contains('open');
+			
+			expect(element.open).toEqual(true);
+			expect(doesNotHaveClassOpenBeforeShow).toEqual(true);
```

Comment:
If we follow the positivity naming rule above, we change this to:
```suggestion
			expect(hasClassOpenBeforeShow).toEqual(false);
```